### ISSUE TITLE
Possible to add multiple Shopify stores?

### ISSUE THREAD

**Issue by benjaminsehl** [davist11/craft-shopify#15]
Hey there,

Wondering if it would be technically possible to add multiple Shopify stores to this plugin, for the case of multiple countries. Any thoughts? Just wondering if you've given it a shot … trying to see what is smartest to do.

Cheers!

**Comment by davist11** [comment]
Hmm so you have multiple stores for different countries? Is localization not something that Shopify has built in?

**Comment by benjaminsehl** [comment]
Unfortunately localization isn’t built in … and to have multiple currencies, you need multiple stores. Not something they’ll be changing either.

**Comment by davist11** [comment]
Are you sure? Seems like multiple currencies is supported based on the docs https://help.shopify.com/themes/customization/currencies

**Comment by benjaminsehl** [comment]
Yep — that will use JS to show the equivalent, but checkout can only happen in one currency per store. I’ve confirmed with the Shopify team. The only current solution native to Shopify is to have multiple stores, with JS-based redirects, which is less than ideal obviously.

**Comment by davist11** [comment]
Doesn't it just convert to the base currency during checkout? Regardless, no, the plugin does not support multiple stores.

**Comment by cliveportman** [comment]
You could create a second installation of the plugin and refactor it a
little. Then use one version of the plugin for one store and the other for
the other store. A bit of a bodge job but it might work.
Clive
